### PR TITLE
Wheel

### PR DESCRIPTION
**Pull request by yiboliu**
### Reasons for making this change
Use wheel as format of uploading to PyPi. 

### Related issues
#2282 

### Checklist

* [x] I've added a screenshot of the changes, if this is a frontend change
* [x] I've added and/or updated tests, if this is a backend change
* [ ] I've run the [pre-commit.sh](https://github.com/codalab/codalab-worksheets/blob/master/pre-commit.sh) script
* [ ] I've updated [docs](https://github.com/codalab/codalab-worksheets/tree/master/docs), if needed

### CODE REVIEW

**Review by nelson-liu — APPROVED** [review]

File: scripts/upload-to-pypi.sh [hunk 1]
```diff
@@ -11,6 +11,6 @@ VER=$1
 set -e
 
 echo "==> Packaging codalab"
-python3 setup.py sdist
+python3 setup.py bdist_wheel
```
Comment (epicfaace) [inline scripts/upload-to-pypi.sh]: Can you ensure we publish _both_ eggs and wheels? It's good to publish source distributions (eggs) as well, see this guide: https://packaging.python.org/guides/distributing-packages-using-setuptools/#packaging-your-project

**Review by epicfaace — CHANGES_REQUESTED** [review]

**Review by epicfaace — COMMENTED** [review]
Can you also double check to ensure that `LICENSE.txt` is included in the built wheel? See https://wheel.readthedocs.io/en/latest/user_guide.html#including-license-files-in-the-generated-wheel-file

File: scripts/upload-to-pypi.sh (hunk 1 above)
Comment (yiboliu) [reply scripts/upload-to-pypi.sh]: Sure. I just pass both argument to it.

**Comment by yiboliu** [comment]
> Can you also double check to ensure that `LICENSE.txt` is included in the built wheel? See https://wheel.readthedocs.io/en/latest/user_guide.html#including-license-files-in-the-generated-wheel-file

image

**Review by epicfaace — APPROVED** [review]

File: setup.cfg [hunk 2]
```diff
@@ -64,3 +64,5 @@ ignore_missing_imports = True
 
 [mypy-urllib3.util.retry]
 ignore_missing_imports = True
+[metadata]
```
Comment (epicfaace) [inline setup.cfg]: We actually don't need this part; it seems that files matching `LICEN[CS]E*` are automatically included: https://wheel.readthedocs.io/en/latest/user_guide.html#including-license-files-in-the-generated-wheel-file

**Pull request merged**